test/CheckoutSdkTest/Payments/AbstractPaymentsIntegrationTest.cs

```diff
@@ -176,5 +138,84 @@ protected async Task<PaymentResponse> Make3dsCardPayment(bool attemptN3d = false
             paymentResponse.ShouldNotBeNull();
             return paymentResponse;
         }
+
+        protected static Phone GetPhone()
+        {
+            return new Phone()
+            {
+                CountryCode = "1",
+                Number = "4155552671"
+            };
+        }
+
+        protected static Address GetAddress()
+        {
+            return new Address()
+            {
+                AddressLine1 = "CheckoutSdk.com",
+                AddressLine2 = "90 Tottenham Court Road",
+                City = "London",
+                State = "London",
+                Zip = "W1T 4TJ",
+                Country = CountryCode.GB
+            };
+        }
+
+        protected static HostedPaymentRequest CreateHostedPaymentRequest(string reference)
+        {
+            return new HostedPaymentRequest()
+            {
+                Amount = 1000L,
+                Reference = reference,
+                Currency = Currency.GBP,
+                Description = "Payment for Gold Necklace",
+                Customer = new CustomerRequest
+                {
+                    Name = "Jack Napier",
+                    Email = GenerateRandomEmail()
+                },
+                ShippingDetails = new ShippingDetails
```

Comment:
Please consider changing using local type inference (var) in all these variables.

Reply:
@rsaestrela I'm not sure if I'm getting you idea.

You mean to create the ShippingDetails like this:
`var ShippingDetails = new ShippingDetails
            {
                Address = GetAddress(),
                Phone = GetPhone()
            };`

and then assign the field link this: `ShippingDetails = ShippingDetails`?